### PR TITLE
#147 added a form to upload a logo

### PR DESCRIPTION
**Pull request by Libbna**
* Added a new menu - Config to admin toolbar's menu. 
* Created a template - config.html.twig
* added a form to upload a logo in the config template. 
* Also created a table config having fields - id and logo (varchar).

SQL file is not updated, create an above table (config) in custom_cms database.

### CODE REVIEW

**Review by RChaubey16 — APPROVED** [review]

File: src/Models/AdminModel.php [hunk 1]
```diff
@@ -61,4 +61,25 @@ public function deleteUser($id){
...
+        $file = $logo['name'];
+        $file_tmp = $logo['tmp_name'];
+
+        $profile_ext = explode('.',  $file);
+        $filecheck = strtolower(end($profile_ext));
+
+        $file_ext_stored = array('jpeg', 'png', 'jpg');
+
+        if (in_array($filecheck, $file_ext_stored)) {
+            $destinationFile = 'assets/images/' . $file;
```
Comment (RChaubey16) [inline src/Models/AdminModel.php:76]: Its storing the values the images folder and not the ' images/logo/ ' folder. Can you just correct the destination path? Rest all looks fine

**Review by RChaubey16 — CHANGES_REQUESTED** [review]
Changes required in AdminModel file.

File: src/Models/AdminModel.php (hunk 1 above)
Comment (Libbna) [reply src/Models/AdminModel.php:76]: I tired that. But the Image is not getting stored in logo folder. Do you know the reason or solution, please share. I will make the required changes then.

Comment (Libbna) [reply src/Models/AdminModel.php]: @RChaubey16  I've updated the changes. Can you please check?

Comment (RChaubey16) [reply src/Models/AdminModel.php]: @Libbna Checked and everything works perfectly. Merging the PR.

**Pull request merged**